app/script_variant.py

```diff
@@ -65,6 +65,40 @@ def romanize_japanese_name_by_name_dict(word):
     return [word]
 
 
+def romanize_japanese_name_by_name_dict(word, is_create=True):
```

Comment:
Add document of is_create argument. Also, it may be better to name it "for_index".